docs/consume-packages/Package-Restore.md

```diff
@@ -87,8 +87,6 @@ Select **Automatically check for missing packages during build in Visual Studio*
 </configuration>
 ```
 
-For non-SDK-style projects, you must select **Allow NuGet to download missing packages** as well as **Automatically check for missing packages during build in Visual Studio** in **Options** to enable automatic restore.
```

Comment:
I'm wondering if this should say:

```suggestion
You must select **Allow NuGet to download missing packages** as well as **Automatically check for missing packages during build in Visual Studio** in **Options** to enable package restore during build.
```

Reply:
Done - github can't commit suggestions for deleted lines so created new commit 🤷‍♂️